src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
+  }
 }
 
+const DAMAGE_AMOUNT = 50;
+
 class Carnivore extends Animal {
-  // write your code here
+  bite(aim) {
+    if (aim instanceof Herbivore && !aim.hidden) {
+      aim.health -= DAMAGE_AMOUNT;
+
+      if (!aim.health) {
+        Animal.alive.splice(Animal.alive.indexOf(aim), 1);
```

Comment:
you could use filter here, like it's suggested in [checklist.md](https://github.com/mate-academy/js_herbivores_and_carnivores/blob/master/checklist.md)
![image](https://github.com/mate-academy/js_herbivores_and_carnivores/assets/147749101/304f6e8f-4aea-4d0e-b150-297793160c51)